src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -132,7 +142,8 @@ const ChatItemMyTextV2 = ({
             previewBackgroundColor="transparent"
             style={{flex: 1}}
             actions={contextMenuActions}
-            onPress={(e) => onContextMenuPressed(e, data, chatType)}>
+            onPress={(e) => onContextMenuPressed(e, data, chatType)}
+            disabled={attachments.length > 1}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `disabled` prop of the `ContextMenu` component is now dependent on whether the length of `attachments` is greater than 1. Ensure that this condition is intended and correctly handles all edge cases.



<!-- This is an auto-generated comment by OSS CodeRabbit -->